src/app/person/person.cpp

```diff
@@ -1,12 +1,14 @@
 #include "person.hpp"
 #include <algorithm>
+#include <iostream>
 #include <numeric>
+#include <stdexcept>
 #include <vector>
 
-Person::Person(const std::string& name, const std::string& surname, const std::string& pesel)
-    : name_{name}, surname_{surname}
+Person::Person(const std::string& name, const std::string& surname, const std::string& pesel, const Gender& gender)
```

Comment:
I saw that you have corrected most occurences of "Gender&", so I feel being a facist for mentioning missing some spots:
1.(here).
2. Other places
./app/receptionist/receptionist.hpp:9:
./app/receptionist/receptionist.hpp:16:     
./app/receptionist/receptionist.cpp:5:                      
./app/receptionist/receptionist.cpp:21:           
./app/patient/patient.cpp:4:
./app/patient/patient.cpp:64:                      
./app/patient/patient.hpp:16:    
./app/patient/patient.hpp:31:                         
./app/person/person.hpp:21:
./app/person/person.cpp:8:
./app/doctor/doctor.hpp:11:
./app/doctor/doctor.hpp:19:                    
./app/doctor/doctor.cpp:6:
./app/doctor/doctor.cpp:27:

Sorry for me being pedantic :(

Reply:
Done